phpBB/phpbb/install/module/requirements/task/check_server_environment.php

```diff
@@ -98,7 +98,7 @@ protected function check_php_version()
 	{
 		$php_version = PHP_VERSION;
 
-		if (version_compare($php_version, '7.1') < 0)
+		if (version_compare($php_version, '7.1.3') < 0)
```

Comment:
This can never happen

Reply:
Fixed

Reply:
I mean that startup.php already check that and do a exit, so that fragment of code is never executed

Reply:
`includes/startup.php` checks for it on `production` or `development`.

`install/startup.php` does not check for it.

Reply:
Also, look at the previous PR: https://github.com/phpbb/phpbb/commit/e76c673e2e6a379ae7c4beccabfaaf8a5eb18371